src/main/java/teammates/common/util/Const.java

```diff
@@ -221,6 +221,8 @@ private Const() {
 
         public static final String COURSE_BACKUP_LOG_MSG = "Recently modified course::";
 
+        public static final String ACCOUNT_ATTRIBUTES_BACKUP_LOG_MSG = "Recently modified account::";
```

Comment:
I think there is not need to extract it to a constant in `Const.java` since it is only used in `AccountAttributes.java`. I think you can also do that for `COURSE_BACKUP_LOG_MSG `.